### PR TITLE
Correct readme file

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Correct readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,17 +28,20 @@ E.g. for scenario 2 following should be used:
...
  1. Build required infrastructure using the configuration from `scenarios/scenario2/` directory
  1. Run `scenario2_setup.yml` playbook from `playbooks/` for created host
 
-There are no credentials stored in `terraform.tfvars` file for the scenario. Recommended way to
-define credentials and overriding is to create some `*.auto.tfvars` file in scenario directory,
-e.g. `secrets.auto.tfvars`. See [variables documentation](https://www.terraform.io/docs/configuration/variables.html) for details.
+There are no credentials stored in `terraform.tfvars` file for the scenario. It is used file called "clouds.yaml" for authentication.
+It must contain credentials and locate in one of the next places:
+ 1) current directory; 
+ 2) ~/.config/openstack;
```
Comment (outcatcher) [inline README.md:40]: Use `code formatting` for paths

File: README.md [hunk 2]
```diff
@@ -20,25 +20,30 @@ Installing terraform for Linux can be done using `install_terraform.sh`
 
 Existing scenario infrastructure build can be triggered using `scenarios/build.sh`
 
-**!NB** Terraform is using OBS for storing remote state
-Following variables have to be set: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`
+**!NB** Terraform is using OBS for storing remote state 
+Following variables have to be set: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `OS_CLOUD` (in case of you have more than one cloud)
```
Comment (outcatcher) [inline README.md:24]: `OS_CLOUD` is not required

File: README.md [hunk 3]
```diff
@@ -20,25 +20,30 @@ Installing terraform for Linux can be done using `install_terraform.sh`
...
 export AWS_ACCESS_KEY_ID=myau_id
 export AWS_SECRET_ACCESS_KEY=sekret_myau
-./build.sh scenario2
+export OS_CLOUD=my_cloud
+ansible-playbook "playbooks/scneario2_setup.yml"
 ```
 This script performs the following actions:
  1. Build required infrastructure using the configuration from `scenarios/scenario2/` directory
- 1. Run `scenario2_setup.yml` playbook from `playbooks/` for created host
+ 2. Run `scenario2_setup.yml` playbook from `playbooks/` for created host
```
Comment (outcatcher) [inline README.md:36]: Markdown automatically increases counter, any number can be used

File: README.md [hunk 4]
```diff
@@ -20,25 +20,30 @@ Installing terraform for Linux can be done using `install_terraform.sh`
...
 
-There are no credentials stored in `terraform.tfvars` file for the scenario. Recommended way to
-define credentials and overriding is to create some `*.auto.tfvars` file in scenario directory,
-e.g. `secrets.auto.tfvars`. See [variables documentation](https://www.terraform.io/docs/configuration/variables.html) for details.
+There are no credentials stored in `terraform.tfvars` file for the scenario. It is used file called "clouds.yaml" for authentication.
+It must contain credentials and locate in one of the next places:
+ 1) `current directory`;
```
Comment (outcatcher) [inline README.md:40]: let's stick to single counter format, `1.`

File: README.md [hunk 5]
```diff
@@ -20,25 +20,30 @@ Installing terraform for Linux can be done using `install_terraform.sh`
 
 Existing scenario infrastructure build can be triggered using `scenarios/build.sh`
 
-**!NB** Terraform is using OBS for storing remote state
+**!NB** Terraform is using OBS for storing remote state
```
Comment (outcatcher) [inline README.md:23]: Trailing space added

File: README.md [hunk 6]
```diff
@@ -20,25 +20,30 @@ Installing terraform for Linux can be done using `install_terraform.sh`
...
+**!NB** Terraform is using OBS for storing remote state 
 Following variables have to be set: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`
 
 E.g. for scenario 2 following should be used:
 ```bash
 cd scenarios
 export AWS_ACCESS_KEY_ID=myau_id
 export AWS_SECRET_ACCESS_KEY=sekret_myau
-./build.sh scenario2
+export OS_CLOUD=my_cloud
```
Comment (outcatcher) [inline README.md:31]: `export OS_CLOUD=my_cloud` can be removed in this example

File: README.md [hunk 7]
```diff
@@ -20,25 +20,30 @@ Installing terraform for Linux can be done using `install_terraform.sh`
...
+ansible-playbook "playbooks/scneario2_setup.yml"
```
Comment (lego963) [inline README.md:32]: is it required to use `""` ?

**Pull request merged**